library.rb

```diff
@@ -0,0 +1,52 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @id = 0
+  end
+
+  attr_accessor :name, :age
+
+  attr_reader :id
```

Comment:
- [ ] [OPTIONAL] I would like to recommend you define getter and setter methods before other functions within a class. This helps maintain a consistent structure and makes the code more readable. 👍 

```
class MyClass
  attr_reader :my_attribute  # Getter method
  
  attr_writer :my_attribute  # Setter method

  def initialize(my_attribute)
    @my_attribute = my_attribute
  end

  def some_function
    # Code for some_function
  end

  def another_function
    # Code for another_function
  end
  
  # More functions...
end

```